inventory/sample-ha.aws.example.com.d/inventory/hosts

```diff
@@ -0,0 +1,40 @@
+#[all:vars]
+# For all group_vars, see ./group_vars/all.yml
+
+# Create an OSEv3 group that contains the master, nodes, etcd, and lb groups.
+# The lb group lets Ansible configure HAProxy as the load balancing solution.
+# Comment lb out if your load balancer is pre-configured.
+[cluster_hosts:children]
+OSEv3
+
+[OSEv3:children]
+masters
+nodes
+etcd
+
+# Set variables common for all OSEv3 hosts
+#[OSEv3:vars]
+
+# For OSEv3 normal group vars, see ./group_vars/OSEv3.yml
+
+# Host Groups
+
+[masters:children]
+tag_group_masters_aws
```

Comment:
Is there a way to avoid the `tag_group` portion of these? I.e.: maybe a setting in the .ini file?

Reply:
Not sure, will investigate that